plugins/browser-plugin-link-click-tracking/src/index.ts

```diff
@@ -210,9 +212,13 @@ function processClick(sourceElement: TrackableElement, includeContent: boolean =
     elementTarget = anchorElement.target;
     elementContent = includeContent ? anchorElement.innerHTML : undefined;
 
+    if (!targetUrl) {
+      console.warn('Link click target URL empty', anchorElement);
```

Comment:
I'm not sure on the semantic difference, but should we be using `_logger?.warn` instead of the `console` directly?